apps/renterd/contexts/config/fieldTips/MaxUploadPrice.tsx

```diff
@@ -31,6 +33,24 @@ export function MaxUploadPriceTips({
 
   return (
     <>
+      {uploadAverage && (
+        <TipNumber
+          type="siacoin"
+          label="Network average"
+          tip="Averages provided by Sia Central."
+          decimalsLimit={0}
+          value={toHastings(uploadAverage)}
+          onClick={() => {
+            formSetField({
+              form,
+              fields,
+              name: 'maxRPCPriceMillion',
```

Comment:
I could be mistaken here but isn't `uploadAverage` used both as `maxUploadPriceTBPinned` and `maxRPCPriceMillion`? edit: yeah looks like you probably want `rpcAverage`

Reply:
oops this whole component is for upload price, this should be `maxUploadPriceTB`. I'll get this fixed.

Reply:
Fixed, and added exhaustive tests for each tip on all price fields.